test/e2e/framework/sessionconfig.go

```diff
@@ -80,6 +81,22 @@ func (cfg SessionConfig) outerHeaderRemoval() *ie.IE {
 	return ie.NewOuterHeaderRemoval(pfcp.OuterHeaderRemoval_GTPUUDPIPV6, 0)
 }
 
+const (
+	BBF_EID                 = 3561
+	BBF_TYPE_APPLY_ACTION   = 15
+	BBF_TYPE_NAT_PORT_BLOCK = 18
+	BBF_APPLY_ACTION_NAT    = 1
+)
+
+func newVendorSpecificStringIE(itype uint16, eid uint16, data string) *ie.IE {
+	newIE := ie.NewVendorSpecificIE(itype, eid, []byte(data))
+	//newIE.Length = 3
```

Comment:
Please remove the commented code.
Maybe just `return ie.NewVendorSpecificIE(itype, eid, []byte(data))`?

Reply:
Ah, sorry, this thing left from a debugging stage. Fixed